docs/scraping-receivers.md

```diff
@@ -0,0 +1,143 @@
+# Scraping metrics receivers
+
+Scraping metrics receivers collect predefined set of metrics from externals sources. Typically, the external source 
+is a monitored entity providing data about itself in some arbitrary format, scraping receivers parse the data and 
+translate it to the OpenTelemetry Pipeline Data. Examples of the scrapings metrics receivers:
+
+- [Host Metrics Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver)
+- [Redis Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver)
+
+## Defining emitted metrics 
+
+Each scraping receiver has `metadata.yaml` file that MUST define all the metrics emitted by the receiver. The
+file is being used to generate an API for metrics recording, user settings to customize the emitted metrics and user
+documentation. The file schema is defined in 
+https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/cmd/mdatagen/metric-metadata.yaml. 
+Defining a metric in `metadata.yaml` DOES NOT guarantee that the metric will always be produced by the receiver. In
+some cases it may be impossible to fetch particular metrics from a system in a particular state. If it's know that 
+the metric is reported only under certain conditions (e.g. a specific version) of the external source, the 
+condition MUST be mentioned in the metric description.
+
+There are two main categories of the metrics emitted by scraping receivers:
+
+- **Default metrics**: emitted by default, but can be disabled in user settings.
```

Comment:
Any thoughts around naming these `Enabled` and `Disabled` metrics? The behaviour between these two is opposite, but the word default doesn't imply it's the opposite of optional

Reply:
I'm not sure. For me, it feels like Default/Optional are more intuitive even if Default doesn't imply it's the opposite of Optional. But I'm good with Enabled/Disabled as well. The only concern is that Enabled is the same name the settings option which may be ok. 

Let's keep it for now, I can rename them if there is no other feedback from @open-telemetry/collector-contrib-approvers

Reply:
Another suggestion: `DefaultOn` / `DefaultOff`. It's more verbose but it makes clear that we are describing the default state only.

`Enabled` is problematic because then when the user disables the metric you have a "disabled Enabled metric"?

`Optional` is also problematic because all metrics are optional.